integration-test/tests/cms/codeofconduct.test.ts

```diff
@@ -0,0 +1,17 @@
+import {expect, test} from '@playwright/test';
+import { codeofconductExepctedInformation } from '../../utils/datafactory/codeofconduct.data';
+test('GET /api/cms/v1/code-of-conduct returns correct team data', async ({request}) => {
+    const response = await request.get(`https://wcc-backend.fly.dev/api/cms/v1/code-of-conduct`);
+ 
+      if (response.status() === 200) {
```

Comment:
Here you should use assert instead of if/else

Reply:
@dricazenck I am using Base URL API_HOST="https://wcc-backend.fly.dev"  in my .env file and all test are failing.

Reply:
@dricazenck - I have done the changes, could you please review again ? Thanks !